mobile/salto/src/routes/+page.svelte

```diff
@@ -102,6 +132,14 @@
       });
     }
 
+    getCurrencies().then((resp) => {
+      if (resp) fiatCurrenciesStored.set(resp);
+    });
```

Comment:
If wallet dont' exist, we should not load this either.
So move it in the previous if statment

Reply:
And it is only used in the settings page.
So call it when the setting page is opened. Not here.